ftest.sh

```diff
@@ -203,7 +203,10 @@ rm -rf $DAOS_BASE/install/tmp
 mkdir -p $DAOS_BASE/install/tmp
 cd $DAOS_BASE
 export CRT_PHY_ADDR_STR=ofi+sockets
-export OFI_INTERFACE=eth0
+
+# Disable OFI_INTERFACE to allow launch.py to pick the fastest interface
+unset OFI_INTERFACE
```

Comment:
Do we want to allow the caller to force it if he wants, to be in parity with allowing `launch.py`'s caller to force it?

```suggestion
```

Reply:
Probably something to address whenever we make changes to `ftest.sh` to allow developers to use it outside of CI.   I chose to use `unset` to avoid conflicts with the likes of:
```
ENV OFI_INTERFACE=eth0
```
[https://github.com/daos-stack/daos/blob/master/utils/docker/Dockerfile.centos.7#L157](url)

Reply:
> changes to ftest.sh to allow developers to use it outside of CI

`ftest.sh` is already usable outside of CI.  I use it all of the time on a few local test nodes.

That said, I do have some additional patch(es) to be smarter about whether NFS mounting the DAOS source dir on all of the nodes is necessary or if it's already done in the environment (i.e. like in the case of auto-mounted /home like on wolf/boro), but that's just a usage enhancement.

> ENV OFI_INTERFACE=eth0
> https://github.com/daos-stack/daos/blob/master/utils/docker/Dockerfile.centos.7#L157

The ENV_OFI_INTERFACE setting in the Dockerfile has no impact on CI testing.  The Dockerfile is not used in test stage of the CI pipeline.

If somebody outside of CI wants to use an interface different than `eth0` with the Dockerfile, they can do so by running `docker` with:

```
docker run -e "OFI_INTERFACE=ib0" ...
```

In any case, I didn't -1 because of this.  Just thought it was worth mentioning so that the setting of OFI_INTERFACE in the environment was consistent (and embracing the principle of least surprise) at all stages of calling the test pipeline.